cloudregister/registerutils.py

```diff
@@ -143,6 +143,25 @@ def clean_cache():
     clean_registered_smt_data_file()
 
 
+# ----------------------------------------------------------------------------
+def clean_standard_credentials():
```

Comment:
nitpicking

the name is a little bit confusing for me

we delete `SCCcredentials` and `NCCcredentials` which I understand as _base_ credentials as stated in the variable name, aren't the repos credentials considered standard (and/but not base) ?

Reply:
I agree, will fix it